### PR TITLE
LPS-66400 Do not retrieve the whole table to calculate if there are rows

### PR DESCRIPTION
**Pull request by mdelapenya**

### CODE REVIEW

**Comment by migue** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by migue** [comment]
@mdelapenya Do you think we really need this? We are using a "cursor" based query, so the prepared statement should take care of everything (it won't retrieve all the rows in the result set).

Are you aware of any driver not doing this right now?

**Comment by mdelapenya** [comment]
Let me try again tomorrow. It took so long with the sample, large database we have
